### ISSUE TITLE
🔧 Issue: "Contributor" Button Not Accessible


### ISSUE THREAD

**Issue by Student-ravneet** [ruchikakengal/WebDevIn100_Days#190]
The "Contributor" button on the homepage is currently not accessible to screen readers. This can create difficulties for users relying on assistive technologies to navigate the website.

I would love to work on it, pls assign this issue to me!!

**Comment by ruchikakengal** [comment]
Okay
